frappe/public/js/frappe/list/bulk_operations.js

```diff
@@ -102,28 +105,25 @@ export default class BulkOperations {
 				pdf_options = JSON.stringify({ "page-size": args.page_size });
 			}
 
-			const w = window.open(
-				"/api/method/frappe.utils.print_format.download_multi_pdf?" +
-					"doctype=" +
-					encodeURIComponent(this.doctype) +
-					"&name=" +
-					encodeURIComponent(json_string) +
-					"&format=" +
-					encodeURIComponent(print_format) +
-					"&no_letterhead=" +
-					(with_letterhead ? "0" : "1") +
-					"&letterhead=" +
-					encodeURIComponent(letterhead) +
-					"&options=" +
-					encodeURIComponent(pdf_options)
-			);
+			const task_id = Math.random().toString(36).slice(-5);
```

Comment:
Shouldn't be set by client side code. Client side code should read task id in response IMO.